### PR TITLE
Added Binary Search and Linear Search to Searching & sorting

### PR DESCRIPTION
**Pull request by a-regular-pear**

### CODE REVIEW

**Comment by fms12** [comment]
it's done. try other one

File: Searching & sorting/Binary Search.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+def binary_search(array, search):
+    low = 0
+    high = len(array) -1
+
```
Comment (fms12) [inline Searching & sorting/Binary Search.py:4]: add a comment for explanation  and try to make a different folder inside the searching & sorting folder

File: Searching & sorting/Linear Search.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+def linearSearch(array, search):
+    for i in range(len(array)):
```
Comment (fms12) [inline Searching & sorting/Linear Search.py:2]: add comment

**Review by fms12 — CHANGES_REQUESTED** [review]

**Comment by fms12** [comment]
mark complete on the issue.

**Comment by a-regular-pear** [comment]
Considering binary search already exists should I mark linear search on the issue?

**Comment by sambhavgupta0705** [comment]
Hey can I add linear search algorithm

**Comment by a-regular-pear** [comment]
I already implemented it in my fork though

**Comment by fms12** [comment]
it's done please check another issue, Thank You for your contribution.

**Pull request closed without merging**